flujo/builtins.py

```
@@ -1318,11 +1318,222 @@ async def welcome_agent(name: str = "Developer") -> str:
+        - kind: step
+          name: init_counter
+          agent: { id: "flujo.builtins.context_set" }
+          input: { path: "scratchpad.counter", value: 0 }
+        ```
+    """
+    from flujo.utils.context import set_nested_context_field
+
+    if context is not None:
+        try:
+            set_nested_context_field(context, path, value)
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)